tests/smartclone_test.go

```diff
@@ -175,15 +164,13 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]SmartClone tests",
 	})
 })
 
-func verifyPVC(dataVolume *cdiv1.DataVolume, f *framework.Framework, testPath string, expectedData string) {
-	hash := md5.Sum([]byte(expectedData))
-	md5sum := hex.EncodeToString(hash[:])
+func verifyPVC(dataVolume *cdiv1.DataVolume, f *framework.Framework, testPath string, md5sum string) {
 	By("verifying pvc was created")
 	targetPvc, err := f.K8sClient.CoreV1().PersistentVolumeClaims(dataVolume.Namespace).Get(context.TODO(), dataVolume.Name, metav1.GetOptions{})
 	Expect(err).ToNot(HaveOccurred())
 
 	By(fmt.Sprint("Verifying target PVC content"))
-	Expect(f.VerifyTargetPVCContentMD5(f.Namespace, targetPvc, testPath, md5sum, int64(len(expectedData)))).To(BeTrue())
+	Expect(f.VerifyTargetPVCContentMD5(f.Namespace, targetPvc, testPath, md5sum, 18874368)).To(BeTrue())
```

Comment:
utils.UploadFileSize?

Reply:
Fixed, I don't like the name of the constant, but we should use it regardless in case we ever rename it.